src/publisher/tests/test_api_v2_views.py

```diff
@@ -251,6 +307,7 @@ def test_article_list(self):
         "a list of published articles are returned to an unauthenticated response"
         resp = self.c.get(reverse("v2:article-list"))
         self.assertEqual(resp.status_code, 200)
+        # TODO: fix
```

Comment:
Are still todo?

Reply:
/sigh 

yes, I suppose I should or it won't ever get done.